src/demo/demo.jsx

```diff
@@ -100,6 +100,87 @@ function handleTabSwitch(target) {
   console.log(`tabChange: ${target.index} - ${target.tabName} - ${target.slug}`);
 }
 
+let footerLinks = [
+  {
+    iconType: `info`,
+    link: `https://example.com`,
+    text: `Info`
+  },
+  {
+    iconType: `email`,
+    link: `https://example.com`,
+    text: `Email`
+  },
+  {
+    iconType: `github`,
+    link: `https://example.com`,
+    text: `Github`
+  },
+  {
+    iconType: `chat`,
+    link: `https://example.com`,
+    text: `IRC`,
+    className: `some-class`
+  },
+  {
+    iconType: `twitter`,
+    link: `https://example.com`,
+    text: `Twitter`,
+    className: `some-class`
+  },
+  {
+    iconType: `facebook`,
+    link: `https://example.com`,
+    text: `Facebook`,
+    className: `some-class`
+  },
+  {
+    iconType: `cc-license`,
+    link: `https://example.com`,
+    text: `License`,
+    className: `some-class`
+  },
+  {
+    iconType: `code-of-conduct`,
+    link: `https://example.com`,
+    text: `Code of Conduct`,
+    className: `some-class`
+  },
+  {
+    iconType: `cookies`,
+    link: `https://www.mozilla.org/privacy/websites/#cookies`,
+    text: `Cookies`,
+    className: `some-class`
+  },
+  {
+    iconType: `legal`,
+    link: `https://example.com`,
+    text: `Legal`,
+    className: `some-class`
+  },
+  {
+    iconType: `privacy`,
+    link: `https://example.com`,
+    text: `Privacy`,
+    className: `some-class`
+  },
+  {
+    iconType: `donate`,
+    link: `https://example.com`,
+    text: `Donate`,
+    className: `some-class`
+  }
+];
+
+let orgs = [
+  {
+    name: `mozilla`,
+    link: `https://mozilla.org`,
+    description: (<p>Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>),
+    className: `mozilla`
```

Comment:
just note: passing `mozilla` as `className` is needed in order for Mozilla wordmark to show up